js1/dom-innerhtml/cvlekce/hod-minci/exercise.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Hod mincí
+demand: 2
+lead: Oživte virtuální minci.
+solutionAccess: lock
+---
+
+Vytvořte si repozitář ze šablony [cviceni-hod-minci](https://github.com/Czechitas-podklady-WEB/cviceni-hod-minci). Repozitář obsahuje stránku s nedokončenou simulací hodu mince. Vaším úkolem bude mincí „hodit“.
+
+1. V souboru `index.js` si do proměnné `padlOrel` uložte hodnotu `true` nebo `false` na základě náhodné hodnoty z `Math.random()`.
```

Comment:
Tady si říkám, jestli ty holky přijdou na to, jak tohle udělat. Přece je to jenom cvičení na DOM a ne na logické myšlení. Takže navrhuju jim to prozradit. Tady například "Pravděpodobnost 50:50 zajistíte porovnáním výsledku funkce s hodnotou 0.5" nebo něco takového.